src/main/kotlin/no/nav/syfo/aktoer/api/AktoerApi.kt

```diff
@@ -6,25 +6,25 @@ import io.ktor.server.response.*
 import io.ktor.server.routing.*
 
 fun Route.registerAktoerApi(aktoerService: AktoerService) {
-    get("/aktoer/hentAktoerIdBySsn/{ssn}") {
+    get("/aktoer/hentAktoerIdByFnr/{fnr}") {
         try {
-            val ssn = call.parameters["ssn"] ?: throw IllegalArgumentException()
-            val identer = aktoerService.getAktoerIdFormSsn(ssn) ?: throw IllegalArgumentException()
+            val fnr = call.parameters["fnr"] ?: throw IllegalArgumentException()
+            val identer = aktoerService.getAktoerIdFormFnr(fnr) ?: throw IllegalArgumentException()
 
             call.respond(identer)
         } catch (e: Exception) {
-            call.respond(HttpStatusCode.NotFound)
+            call.respond(HttpStatusCode.NotFound, "Could not find user")
         }
     }
 
-    get("/aktoer/hentSsnByAktoerId/{aktoerId}") {
+    get("/aktoer/hentFnrByAktoerId/{aktoerId}") {
```

Comment:
Ble mye bedre med Fnr i stedet for Ssn overalt :) - domeneord er ❤️!

Reply:
Enig, med June! Når man først endrer navnet på disse, kunne man kanskje vurdert å endre fra `aktoerId` til `aktorId` også?